src/main.rs

```diff
@@ -12,6 +12,7 @@ use std::error::Error;
 use std::ffi::OsStr;
 use std::fs;
 use std::io::Write;
+#[cfg(target_os = "linux")]
```

Comment:
[Doesn't look like it](https://stackoverflow.com/a/41743950/1858225); this should probably be something like `target_family = "unix"`.